src/test/groovy/com/benjaminsproule/swagger/gradleplugin/JaxrsPluginTest.groovy

```diff
@@ -62,10 +64,56 @@ class JaxrsPluginTest {
         project.tasks.generateSwaggerDocumentation.execute()
 
         def swaggerFile = new File("${expectedSwaggerDirectory}/swagger.json")
-        assertSwaggerJson(swaggerFile)
+        assertSwaggerJson(swaggerFile, 'string')
     }
 
-    private static void assertSwaggerJson(File swaggerJsonFile) {
+    @Test
+    @Ignore
+    /* this test is ignored because it adversely impacts the spring MVC plugin test, it is fine when run in isolation,
+    but as part of a suite, once a model modifier is registered in EnvironmentConfigurer by calling
+    ModelConverters.getInstance().addConverter(modelModifier), it is not possible to remove or clear it again unless
```

Comment:
Is it not possible to access `removeConverter`?

Reply:
Ah that didn't even occur to me! Ill add that in and remove the comments. Thanks.